sys/net/gnrc/sock/ip/gnrc_sock_ip.c

```diff
@@ -36,7 +36,7 @@ int sock_ip_create(sock_ip_t *sock, const sock_ip_ep_t *local,
         (local->netif != remote->netif)) {
         return -EINVAL;
     }
-    memset(&sock->local, 0, sizeof(sock_ip_t));
+    memset(&sock->local, 0, sizeof(sock_ip_ep_t));
```

Comment:
I have mixed feelings whether to use the type or the instance as a parameter for `sizeof` here. I understand @keestux's rationale, but I also think that using the type makes the developer aware of how much memory is actually initialized/copied.

However, I totally agree with @miri64 that this is out of scope for this PR.